<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManager</class>
 <widget class="QWidget" name="UserManager">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>状态：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxStatus">
       <item>
        <property name="text">
         <string>全部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>待审批</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>已审批</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>已拒绝</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxEdit">
     <property name="title">
      <string>用户编辑</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelRole">
        <property name="text">
         <string>角色：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxRole">
        <item>
         <property name="text">
          <string>工程师</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>管理员</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="text">
        <string>修改角色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApprove">
       <property name="text">
        <string>审批通过</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReject">
       <property name="text">
        <string>拒绝</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 